Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,44 @@
+//1. Declare a variable on each type using two options: plain creation, creation with constructor.
+let numberVar = 32;
+let stringVar = 'Product Owner';
+let booleanVar = true;
+let nullVar = null;
+let undefinedVar; 
+let objectVar = { a: 'prior experience', b: 'resume'};
+let bigIntVar = BigInt(9007199254740991);
+
+console.log(stringVar,numberVar,booleanVar,nullVar,undefinedVar,objectVar,bigIntVar);
+
+//2. Use operator typeof on each of them.
+
+console.log(typeof numberVar); 
+console.log(typeof stringVar);
+console.log(typeof booleanVar);
+console.log(typeof nullVar); 
+console.log(typeof undefinedVar);
+console.log(typeof objectVar);
+console.log(typeof bigIntVar);
+
+//3. Demonstrate object's references.
+
+objectValue = { age: 32, position: 'product owner', priorExperience: true };
+objectValue.age = 34;
```

Comment:
by object references meant to demonstrate that variable contains just reference to object in memory and when you assign this variable to other variable they will affect each other on change.